clusters/dawntest/configmap.yaml

```diff
@@ -34,6 +34,28 @@ data:
         machineFlavor: "bm.xe9640.xp8468.m1024.pvc.gpu"
         machineCount: 2
         kubeadmConfigSpec:
+          preKubeadmCommands:
+            - |
+              # Adapted from https://dgpu-docs.intel.com/driver/installation.html#ubuntu
```

Comment:
Can we try add a quick check to skip on the case where there are no GPU nodes installed, e.g. `lspci | grep "Ponte"`?